src/textual/css/_help_text.py

```diff
@@ -158,7 +184,7 @@ def spacing_wrong_number_of_values(
     )
 
 
-def spacing_invalid_value(
+def spacing_invalid_value_help_text(
```

Comment:
While I was exploring this "HelpText factories" machinery I noticed that contrary to the others the first two ones didn't have the `help_text` suffix - for teh sake of consistency, I added it :slightly_smiling_face: